uui/components/datePickers/__tests__/RangeDatePicker.test.tsx

```diff
@@ -13,21 +13,20 @@ async function setupRangeDatePicker(params: { value: { from: string; to: string
             value: value ?? { from: null, to: null },
```

Comment:
Let's also add tests for:
 - entering date from keyboard in target input. It will also support entering date not only in default format, but also in some others supported by DatePicker formats.
 - moving focus in inputs depends on which part are selecting now
 - onFocus/onBlur props
 - isHoliday prop
 - onOpenChange prop

Reply:
Still doesn't find test for `onOpenChange` prop and 'moving focus in inputs depends on which part are selecting now' depends on selection in picker body, not only in cases when some input was clicked directly.

Reply:
Added tests for `focus`change.